### PR TITLE
Added August's Organisers' Two-fer

### PR DESCRIPTION
**Pull request by yanneves**

### CODE REVIEW

File: _posts/2023-08-03-organisers-two-fer-typescript-tailwind.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+#### Picking up Tailwind
+
+Yann will share a lightning talk on upcoming features in Tailwind that simplify the developer experience and progressively enhance new CSS concepts. Try to `@container` your excitement.
+
+This session will explore advanced concepts and assumes a working knowledge of either or both tools. All experience levels are welcome as always, but this will be a chance to nerd out if your `:nth-of-type()` is type safe recursion for breakfast.
+
+---
+
+You will be able to join us in-person at [Runway East](https://runwayea.st/locations/brighton?utm_source=external&utm_medium=event&utm_campaign=sponsorship) or [online](https://www.youtube.com/watch?v=---).
```
Comment (yanneves) [inline _posts/2023-08-03-organisers-two-fer-typescript-tailwind.md:47]: needs livestream link

File: _posts/2023-08-03-organisers-two-fer-typescript-tailwind.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+date: 2023-08-03T19:15:00
+meetup: https://www.meetup.com/async-web-tech-meetup/events/295080379/
+speakers:
+  - name: Jake 'Sid' Smith
+    link: https://twitter.com/JakeSidSmith
+  - name: Yann Eves
+    link: https://yannev.es
+sponsors:
+image:
+  url:
```
Comment (yanneves) [inline _posts/2023-08-03-organisers-two-fer-typescript-tailwind.md:13]: needs an image

File: _posts/2023-08-03-organisers-two-fer-typescript-tailwind.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  name: Runway East
+  link: https://runwayea.st/locations/brighton?utm_source=external&utm_medium=event&utm_campaign=sponsorship
+  location: https://goo.gl/maps/WGe2p9D7Wup3LdNt8
+  address: York And Elder Works, 50 New England St, Brighton BN1 4AW
+  latlong: 50.8325788,-0.1420808
+layout: event.html
+collection: events
+---
+
+**NEW VENUE:** We're trialling a move to [Runway East](https://runwayea.st/locations/brighton?utm_source=external&utm_medium=event&utm_campaign=sponsorship), a brand new space we think makes an even better long-term home for Async meetups. Located a short walk from Brighton railway station on the ground floor of the new York & Elder development.
```
Comment (JakeSidSmith) [inline _posts/2023-08-03-organisers-two-fer-typescript-tailwind.md:31]:
```suggestion
**NEW VENUE:** We're trialling a move to [Runway East](https://runwayea.st/locations/brighton?utm_source=external&utm_medium=event&utm_campaign=sponsorship), a brand new space we think makes an even better long-term home for Async meetups. Located [a short walk](https://goo.gl/maps/oKjqEjpyzC1sLKLp7) from Brighton railway station on the ground floor of the new York & Elder development.
```

File: _posts/2023-08-03-organisers-two-fer-typescript-tailwind.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+This month we bring you a special organisers' two-fer presented by Jake and Yann!
+
+#### The Power of TypeScript
+
+Jake will speak on advanced patterns in TypeScript you'll be glad to know. Along with powerful uses you may wish you didn't see. `<T>`rigger warning.
+
+#### Picking up Tailwind
```
Comment (JakeSidSmith) [inline _posts/2023-08-03-organisers-two-fer-typescript-tailwind.md:39]: Would this read better having the lightning talk first? (assuming you'll be speaking first)

**Review by JakeSidSmith — COMMENTED** [review]
Will get you an image for this in a sec.

File: _posts/2023-08-03-organisers-two-fer-typescript-tailwind.md (hunk 1 above)
Comment (JakeSidSmith) [reply _posts/2023-08-03-organisers-two-fer-typescript-tailwind.md]: Have removed the link for now. We can add this at a later time.

**Pull request merged**